docs/releases/2.1.txt

```diff
@@ -201,6 +201,9 @@ Models
 
 *  :meth:`.QuerySet.raw` now supports :meth:`~.QuerySet.prefetch_related`.
 
+* :class:`~django.contrib.postgres.search.SearchQuery` now produces a more
```

Comment:
is it a bug fix for 2.1 or new feature for 2.2?

Reply:
It's marked as a bugfix in the bugtracker. I'll change the notes